code/style.css

```diff
@@ -1,58 +1,249 @@
+/*HEADER, BODY AND CARDS SECTION*/
+
 body {
   margin: 0;
-  font-family: "Courier New", Courier, monospace;
+  font-family: Girassol;
+  background-image: url(./castleroom.jpg);
```

Comment:
It's far more safe approach to wrap path in parentheses. Without them, some browsers might not find the path
`background-image: url("./castleroom.jpg");`